### DISCUSSION TITLE
关于功能“评论区IP属地显示”的文案改进建议

### DISCUSSION THREAD

**Discussion by RingingVolcano** [the1812/Bilibili-Evolved#4490] [功能建议 / Ideas]
![89b1e342cdd6c902c3586a73c698c9ac](https://github.com/the1812/Bilibili-Evolved/assets/72140353/fea038ef-019f-4fcc-b367-e20cc899ab31)
@LightQuanta 艾特一下作者
时间后接的 “IP属地：天津” 略有突兀且文字较长，建议可考虑精简一下改成 “发布于 天津” 、 “来自 天津” 或者直接写成 “天津”
虽然IP属地外显是政策的强制要求但也可以把文案做得更柔和一些 参照小红书帖子下的显示方法

此外，请问可以给网页端的个人主页、动态和视频发布地也都添加IP属地外显，技术上能否实现？

**Comment by Geeyun-JY3** [comment]
手机客户端就是这种格式吧。

**Comment by LightQuanta** [comment]
`IP属地：xxx`这个其实是b站官方的接口返回的格式，返回的内容直接就是`"location": "IP属地：xxxx"`

包括这个显示效果其实也是一样，我是在还原b站原先设想的样式。官方以前是有打算实装网页端评论属地显示的，但不知为何没有做，详见[这里](https://lq0.tech/2023/02/27/bilicommentreverse/#%E6%97%A7%E7%89%88%E8%AF%84%E8%AE%BA%E5%8C%BA%E9%80%86%E5%90%91)

你的需求的话，我考虑之后加个设置界面进行设置，比如设置显示位置、自定义显示格式等

个人主页、动态、视频发布（这个其实也是一条动态）这几个是确实做不到，官方的接口直接没有返回IP属地。我估计真要实现得调用移动端的gRPC接口才行了，这样怕是还得整额外的登录什么的，不考虑实装

**Reply by RingingVolcano** [reply]
好的，谢谢！
那之后在已有界面加个显示格式设置吧 因为官方这个确实不太美观
